engine/src/core/app/entry_point.h

```diff
@@ -20,10 +20,10 @@ int main() {
     delete application;
 }
 
-#elif
+#else
 
 int main() {
```

Comment:
I think we should delete this until it will be necessary.

Reply:
This is a fix for master branch, since it is messed up right now.